java/src/main/java/vowpalWabbit/responses/ActionProb.java

```diff
@@ -0,0 +1,42 @@
+package vowpalWabbit.responses;
+
+/**
+ * Created by jmorra on 8/12/16.
+ */
+public class ActionProb {
+
+    private final int action;
```

Comment:
Yea, I'm gonna leave it like this cause it's more Java-y (although it's super verbose).